manifests/params.pp

```diff
@@ -35,6 +40,7 @@
   $log_dir  = '/var/log/candlepin'
 
   $crl_file = '/var/lib/candlepin/candlepin-crl.crl'
+  $ca_crt = undef
```

Comment:
I am not seeing this used anywhere? Or is it a typo and you meant https://github.com/Katello/puppet-candlepin/pull/32/files#diff-60ae41fd0a31977447947f59940ee9a4R140